### PR TITLE
Don't cache generically-handled paths

### PR DESCRIPTION
**Pull request by jranson**

### CODE REVIEW

**Comment by jranson** [comment]
Apologies for all the apparent changes on the Test file, it looks like my Textmate formatter is different than what Phil uses.  The only functional change to tests is the removal of TestTricksterHandler_promAPIProxyHandler since that func was subsumed by promFullProxyHandler.

File: handlers_test.go [hunk 1]
```diff
@@ -14,25 +14,25 @@
 package main
 
 import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"github.com/go-kit/kit/log"
```
Comment (SuperQ) [inline handlers_test.go]: We usually sort external imports from stdlib imports by putting them at the end with a newline between.

**Comment by SuperQ** [comment]
@jranson I think the issue is missing `go fmt`.  If you want, I can clean that up in a separate PR.

**Comment by SuperQ** [comment]
https://github.com/Comcast/trickster/pull/69 should cleanup the code issues.

File: handlers_test.go (hunk 1 above)
Comment (jranson) [reply handlers_test.go]: Looks like you picked this one up in #69. I will merge #68 first once it gets the thumbs up.

**Pull request merged**